### ISSUE TITLE
[00001] tt-train tests: 19 tests failed with numerical accuracy assertion failures. SDPABackwardTest: xt::all(xt::isfinite(composite_dV)) = false (non-finite values detected). Failed tests include: CrossEntropyForwardTest, LinearOpTest, SoftmaxTest,...

### ISSUE THREAD

**Issue by ebanerjeeTT** [tenstorrent/tt-auto-triage#2160]
**Number of Occurrences:** 1

## Error Message

```
tt-train tests: 19 tests failed with numerical accuracy assertion failures. SDPABackwardTest: xt::all(xt::isfinite(composite_dV)) = false (non-finite values detected). Failed tests include: CrossEntropyForwardTest, LinearOpTest, SoftmaxTest, SDPAForwardTest, SDPABackwardTest, AdamWComparisonTest.
```

## All Occurrences
This error has occurred 1 time(s):

1. [[CENTROID] June 1st, 12:34pm, 46.26 seconds (marked as ND)](https://github.com/tenstorrent/tt-metal/actions/runs/26750102721/job/78845600247) - merge-gate / Merge Gate Status (commit: bf2870164f6ce3c6410ecede4bf29215b0923860)

    ```
    tt-train tests: 19 tests failed with numerical accuracy assertion failures. SDPABackwardTest: xt::all(xt::isfinite(composite_dV)) = false (non-finite values detected). Failed tests include: CrossEntropyForwardTest, LinearOpTest, SoftmaxTest, SDPAForwardTest, SDPABackwardTest, AdamWComparisonTest.
    ```
